users/views.py

```diff
@@ -1,3 +1,34 @@
-from django.shortcuts import render
+import requests, jwt
 
-# Create your views here.
+from django.http      import JsonResponse
+from django.views     import View
+
+from config.settings  import SECRET_KEY, ALGORITHM
+from users.models     import User
+
+class KakaoSignInView(View):
+    def get(self, request):
+        try:
+            kakao_access_token = request.headers["Authorization"]
+            user_information   = requests.get("https://kapi.kakao.com/v2/user/me", headers=({"Authorization": f"Bearer {kakao_access_token}"})).json()
+
+            if user_information.get("code") == -401 :
+                return JsonResponse({"MESSAGE" : "INVALID_TOKEN"}, status=401)
+
+            email             = user_information["kakao_account"]["email"]
+            nickname          = user_information["kakao_account"]["profile"]["nickname"]
+            profile_image_url = user_information["kakao_account"]["profile"]["profile_image_url"]
+            login_platform    = "kakao"
+
+            user, create = User.objects.get_or_create(
+                email             = email,
+                nickname          = nickname,
+                profile_image_url = profile_image_url,
+                login_platform    = login_platform
+            )
```

Comment:
get_or_create을 사용 할 때는 확실히 데이터를 특정 지을 수 있는 값만 사용해야 합니다.(ex pk) 카카오 로그인 경우에는 kakao_id가 이에 해당합니다. create 할 때 추가 되어야 하는 부수적인 데이터는 default 옵션을 사용해 주세요. defualt 옵션은 create 되는 경우에만 디비에 적용되는 값입니다.
```suggestion
            user, create = User.objects.get_or_create(
                kakao_id = kakao_id,
                default = {
                    email             = email,
                    nickname          = nickname,
                    profile_image_url = profile_image_url,
                    login_platform    = login_platform
                }
            )
```

Reply:
해당 피드백 역시 db에 kakao_id 필드 추가 후 반영하도록 하겠습니다!